addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -1,123 +1,34 @@
-        }
-        background-color: $o-view-background-color;
-        width: auto;  // bootstrap override
-
-        // Inform the user that he may click on a cell to be redirected to a list view of the
-        // items corresponding to the clicked cell
-        &.o_enable_linking {
-            .o_pivot_cell_value:not(.o_empty):hover {
-                color: $o-brand-primary;
-                cursor: pointer;
-
-            }
-        }
-
-        thead th:not(.o_pivot_header_cell_closed):not(.o_pivot_header_cell_opened):not(.o_pivot_header_cell) {
-            text-align: center;
-        }
-
-        th, td {
-            border-color: gray('200');
-            border-width: $border-width $border-width $border-width $border-width;
-        }
-
-        th {
-            font-weight: $font-weight-normal;  // bootstrap override
-            background-color: gray('100');
-        }
-
-        @mixin o-pivot-header-cell {
-            cursor: pointer;
-            white-space: nowrap;
-            user-select: none;
-            &:hover {
-                background-color: gray('200');
-                color: $headings-color;
-            }
-        }
-
-        .o_pivot_measure_row, .o_pivot_origin_row {
-            @include o-pivot-header-cell;
-        }
-
-        .o_pivot_header_cell {
-            white-space: nowrap;
-            user-select: none;
-        }
-
-        .o_pivot_header_cell_closed {
-            @include o-pivot-header-cell;
-            &::after, &::before {
-                font-family: FontAwesome;
-                margin-right: 8px;
-            }
-            &::after {
-                content: #{"/*!rtl:''*/"} none;
-            }
-            &::before {
-                content: #{"/*!rtl:none*/"} "";
-            }
-        }
-
-        .o_pivot_header_cell_opened {
-            @include o-pivot-header-cell;
-            &::after, &::before {
-                font-family: FontAwesome;
-                margin-right: 8px;
-            }
-            &::after {
-                content: #{"/*!rtl:''*/"} none;
-            }
-            &::before {
-                content: #{"/*!rtl:none*/"} "";
-            }
-        }
+    // This can be removed after the migration to BS5.
+    th, td {
+        border-width: $border-width;
+    }
 
-        .o_pivot_sort_order_asc::after {
-            @include o-caret-up;
-            margin-left: 5px;
-        }
+    .o_pivot_measure_row,
+    .o_pivot_origin_row,
+    .o_pivot_header_cell_closed,
+    .o_pivot_header_cell_opened {
+        // "user-select-none" class doesn't exist in BS 4.3.1, the user-select property
+        // could be replaced by a class after the migration to BS5
+        user-select: none;
+        @include o-hover-text-color($body-color, $headings-color);
 
-        .o_pivot_sort_order_desc::after {
-            @include o-caret-down;
-            margin-left: 5px;
+        &:hover {
+            background-color: gray('200') !important;
```

Comment:
I found +20 occurrences in the entire UI.
You can keep it like that for now, but we should consider adding specific utility classes, like `bg-hover-200`or smthg.
to discuss...